### ISSUE TITLE
Pan Smoothness changes depending on zoom <> rotation

### ISSUE THREAD

**Issue by marwie** [yomotsu/camera-controls#421]
### Describe the bug

Depending on if I use the scrollwheel or rotate with the left button the panning smoothness seems to change

Using latest cc: "camera-controls": "^2.3.7"

### To Reproduce

1) Visit the live sample 
2) pan with the right mouse button, notice it's pretty smooth
3) zoom with the mouse wheel
4) pan with the right mouse button, notice it's now much faster

### Code

```js
this._orbit.mouseButtons.left = CameraControls.ACTION.ROTATE;
            this._orbit.mouseButtons.right = CameraControls.ACTION.TRUCK;
            this._orbit.mouseButtons.wheel = CameraControls.ACTION.DOLLY;
            this._orbit.touches.one = CameraControls.ACTION.TOUCH_ROTATE;
            this._orbit.touches.two = CameraControls.ACTION.TOUCH_DOLLY_TRUCK;
            this._orbit.touches.three = CameraControls.ACTION.TOUCH_DOLLY_TRUCK;

this is set once after creating the controls:
        this._orbit.minDistance = this.minimalZoom;
        this._orbit.maxDistance = this.maximalZoom;
        this._orbit.smoothTime = 1;
        this._orbit.draggingSmoothTime = .3;
        this._orbit.dollySpeed = 1;
        this._orbit.dollyToCursor = true;
```

### Live example

https://app.songsofcultures.com/?lang=en&scene=water-ferns

### Expected behavior

Pan smoothness stays as 

### Screenshots or Video

https://github.com/yomotsu/camera-controls/assets/5083203/1c50fe72-b402-4338-bbea-9572f2a0f50a

### Device

Desktop

### OS

Windows

### Browser

Chrome

**Comment by yomotsu** [comment]
Thanks for the report again.
I think I managed it with PR https://github.com/yomotsu/camera-controls/pull/422
And will merge tonight

**Comment by marwie** [comment]
Sounds good - thank you!

**Comment by yomotsu** [comment]
I think this has been solved by merging PR #422.
Let me close the issue.
